reporter/internal/pdata/generate.go

```diff
@@ -14,7 +14,7 @@ import (
 	"go.opentelemetry.io/collector/pdata/pprofile"
 	"go.opentelemetry.io/otel/attribute"
 
-	semconv "go.opentelemetry.io/otel/semconv/v1.34.0"
+	semconv "go.opentelemetry.io/otel/semconv/v1.36.0"
```

Comment:
Why not update to the latest version?

```suggestion
	semconv "go.opentelemetry.io/otel/semconv/v1.37.0"
```